src/firmware_new/boards/frankie_v1/main.c

```diff
@@ -150,7 +170,7 @@ int main(void)
     MX_TIM4_Init();
     /* USER CODE BEGIN 2 */
 
-    initWheelMotors();
+    initIoLayer();
```

Comment:
Maybe specify this is motor IO layer explicitly

Reply:
Hmm, the intent is to eventually have this init all the IO stuff as we add it. I've moved the drivetrain initialization stuff into it's own function and called that from `initIoLayer`, does that help?

Reply:
Yep :+1: